webservices/rest.py

```diff
@@ -192,7 +192,8 @@ def add_aggregate_resource(api, view, schedule, label):
 api.add_resource(candidate_aggregates.ScheduleABySizeCandidateView, '/schedules/schedule_a/by_size/by_candidate/')
 api.add_resource(candidate_aggregates.ScheduleAByStateCandidateView, '/schedules/schedule_a/by_state/by_candidate/')
 
-api.add_resource(candidate_aggregates.TotalsCandidateView, '/candidate/<candidate_id>/totals/')
```

Comment:
Can we keep this url and direct to the TotalsCandidateView? I feel like this would be a breaking change that we don't need to make.